site/source/cookbook/widgets/i18n-use-localization.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: docs
+category: widgets
+title: Handle string localization within a widget
+overview: Use a localized message within a widget
+---
+
+## Objective
+
+This recipe demonstrates how to set up internationalization within your Dojo 2 app.
+
+## Procedure
+
+1. Your localized strings must be part of your app bundle. To achieve this, first add the following configuration to the file `.dojorc`:
+
+```json
+{
+    "build-webpack": {
+        "locale": "en",
+        "supportedLocales": [ "FR", "ES" ],
+        "messageBundles": [ "src/nls/common" ]
+    }
+}
+```
+
+The `supportedLocales` property value must be an array of all supported languages.
+
+2. The bundles directory (`messageBundles`) you specified in step one must be created.
+
+* Create folder: `src/nls`
```

Comment:
directory instead of folder please (and on the next two lines)

Reply:
Updated in those 3 places